source/aks/rebuilding-aks-clusters.html.md.erb

```diff
@@ -46,6 +46,7 @@ Click on **Run** to start the pipeline.
 - Change Jenkins to use the other cluster, e.g. [cnp-flux-config#19886](https://github.com/hmcts/cnp-flux-config/pull/19886/files) .
   - After merging above PR, if Jenkins is not updated [Check Here](https://build.platform.hmcts.net/configure), you MUST manually change Jenkins config to use the other cluster.
 - Swap external DNS active cluster, e.g. [cnp-flux-config#19889](https://github.com/hmcts/cnp-flux-config/pull/19889) 
```

Comment:
Can we also explicitly state at the start of this line "if it is a preview environment" in the same way?